### PR TITLE
Using symlinks on Windows to compile containers-tests

### PR DESCRIPTION
**Pull request by autonym8**
Added instructions to compile containers-test using symbolic links for Windows users.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,6 +28,15 @@ cabal test containers-test
...
 ```
 
+#### For Windows users : 
+
+To compile `containers-test`, you need symbolic links to be activated on git.
+To do so on Windows 10 on higher, follow these steps :
+
+1. Activate developper mode on in your Windows preferences
+2. Enable git symlinks : `git config --global core.symlinks true`
+3. Clone the repository again once git is properly configured
```
Comment (konsumlamm) [inline CONTRIBUTING.md:38]:
```suggestion
To compile `containers-test`, you need symbolic links to be activated on git.
To do so on Windows 10 or higher, follow these steps:

1. Activate developer mode in your Windows preferences
2. Enable git symlinks: `git config --global core.symlinks true`
3. Clone the repository again once git is properly configured
```

Comment (treeowl) [reply CONTRIBUTING.md:38]: Let's put this in a separate Windows section so people running Linux, OSX, etc., don't get confused.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -28,6 +28,15 @@ cabal test containers-test
 cabal bench containers-test
 ```
 
+#### For Windows users :
```
Comment (treeowl) [inline CONTRIBUTING.md:31]:
```suggestion
#### For Windows users
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -28,6 +28,15 @@ cabal test containers-test
...
 
+#### For Windows users : 
+
+To compile `containers-test`, you need symbolic links to be activated on git.
+To do so on Windows 10 or higher, follow these steps:
+
+1. Activate developer mode in your Windows preferences
+2. Enable git symlinks: `git config --global core.symlinks true`
+3. Clone the repository again once git is properly configured
+
```
Comment (treeowl) [inline CONTRIBUTING.md:39]: Could you put a period at the end of each step?

**Comment by treeowl** [comment]
Awesome! Thanks a lot!

**Comment by autonym8** [comment]
You're welcome, I was happy to help

**Pull request merged**